apps/mtaos/README.md

```diff
@@ -8,6 +8,7 @@ A Helm chart for deploying the MTAOS CSC.
 |-----|------|---------|-------------|
 | affinity | object | `{}` | This specifies the scheduling constraints of the pod |
 | annotations | object | `{}` | This allows the specification of pod annotations |
+| butlerSecret | string | `nil` | This key allows for specification of Butler secret information If this section is used, it must contain the following attribute: __containerPath__ (The directory location for the Butler secret) __dbUser__ (The username for the Butler backend database) |
```

Comment:
You seen to be mixing __ (like this line) and _ (like line 14). Please choose one and stick to that.

Reply:
Fixed.